app/assets/javascripts/discourse/app/initializers/subscribe-user-changes.js

```diff
@@ -1,15 +0,0 @@
-export default {
-  name: "subscribe-user-changes",
-  after: "message-bus",
-
-  initialize(container) {
-    const user = container.lookup("current-user:main");
-
-    if (user) {
-      const bus = container.lookup("message-bus:main");
-      bus.subscribe("/user", (data) => {
```

Comment:
The channel doesn't need to be so general when it is only used for drafts count.